routes/albums.js

```diff
@@ -0,0 +1,109 @@
+//Making request through node
+var Request = require('request');
+var Joi = require('joi');
+var Auth = require('./auth');
+var isOwner = require('./is-owner');
+exports.register = function(server, options, next) {
+  server.route([
+    {
+      method: 'GET',
+      path:'/api/v1/albums',
+      handler: function(request, reply) {
+        var db = request.server.plugins['hapi-mongodb'].db;
+        db.collection('albums').find().toArray(function(err, albums) {
+          if (err) { return reply(err);}
+          reply(albums);
+        });
+      }
+    },
+    {
+      method: 'POST',
+      path: '/api/v1/albums',
+      config: {
+        handler: function(request, reply) {
+          var db = request.server.plugins['hapi-mongodb'].db;
+          var album = request.payload.album;
+          //check if authenticated
+          Auth.authenticated(request, function(result) {
+            if (!result.authenticated) {
+              reply(result);
+            }
+            var user_id = request.session.get('slidshw_session').user_id;
+            album['user_id'] = user_id;
```

Comment:
You can use dot `.` notation all the way if you want. ex. ` album.user_id = user_id`